api/omb_eregs/settings.py

```diff
@@ -188,6 +188,9 @@
 
 STATIC_URL = '/static/'
 STATIC_ROOT = os.environ.get('TMPDIR', '.') + '/static/'
+STATICFILES_DIRS = [
+    os.path.join(BASE_DIR, 'webpack-static'),
+]
```

Comment:
Since the webpack-built stuff can apply to any Django app, I thought it might be best to just have a separate staticfiles directory for all the webpack stuff.  We might be able to configure webpack to output the different bundles into the `static` directories of the respective apps that they belong to, though, if that's preferable.